tools/publish-jar-local.sh

```diff
@@ -0,0 +1,52 @@
+set -euo pipefail
+
+if [[ -f "$HOME/.jabba/jabba.sh" ]]; then
+  echo "Found jabba"
+  source "$HOME/.jabba/jabba.sh"
+  jabba use adopt@1.8.0-292
+fi
+
+JAVA_VERSION=`java -version 2>&1 | awk -F '"' '{print $2}'`
+if [[ $JAVA_VERSION != "1.8.0_292" ]]; then
+  echo "Wrong java version ! Required 1.8.0_292 but found $JAVA_VERSION"
+  exit 42
+fi
+
+# This script works with a directory name as first parameter
+# You need to have the libcore already built in this directory with JNI enabled
+# e.g.:
+# cd lib_build_dir_jni
+# cmake .. -DSYS_OPENSSL=ON -DOPENSSL_USE_STATIC_LIBS=TRUE -DTARGET_JNI=ON -DPG_SUPPORT=ON
+# cmake --build . --parallel
+#
+# And then you call the script:
+# ./publish-jar-local.sh ./lib_build_dir_jni
+LIB_BUILD_DIR=$1
```

Comment:
you should make sure `$1` is not empty

Reply:
`set -u` at the beginning of the scripts ensures this

Reply:
TIL